source/common/http/conn_manager_impl.cc

```diff
@@ -617,14 +617,8 @@ void ConnectionManagerImpl::ActiveStream::decodeHeaders(HeaderMapPtr&& headers,
     }
   }
 
-  // Check for maximum incoming header size. Both codecs have some amount of checking for maximum
-  // header size. For HTTP/1.1 the entire headers data has be less than ~80K (hard coded in
-  // http_parser). For HTTP/2 the default allowed header block length is 64k.
-  // In order to have generally uniform behavior we also check total header size here and keep it
```

Comment:
I had moved these comments to the http connection manager .proto, and just added back more of the existing details from this deletion. Is it ok for this info to sit in the api/proto rather than conn manager?